hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/HFileCleaner.java

```diff
@@ -17,22 +17,32 @@
  */
 package org.apache.hadoop.hbase.master.cleaner;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
It's weird that this wants it all in one block... but if checkstyle likes it who am I to disagree? :)